src/app/analyze/analyze.module.ts

```diff
@@ -7,9 +7,15 @@ import { AnalyzeRoutingModule } from './analyze-routing.module';
 
 import { AnalyzeOverviewModule } from './analyze-overview/analyze-overview.module';
 
+import {RenderStackDetailsComponent} from './stackreports/render-stack-details/render-stack-details.component';
+import {StackModule} from './stackreports/render-stack-details/module';
+
+import {StackRecoModule} from './stackreports/stack-recommendation/module';
```

Comment:
Don't use abbreviations in names. Copy the naming approach I did in the stack-details module for file names.